app.js

```diff
@@ -221,22 +219,22 @@ function createServer(app) {
 
 }
 
-
 /**
  * The service's entry point. It takes over the configuration
  * options and the logger- and metrics-reporting objects from
  * service-runner and starts an HTTP server, attaching the application
  * object to it.
+ * @param {Object} options the options to initialise the app with
+ * @return {bluebird} HTTP server
  */
-module.exports = function(options) {
+module.exports = function (options) {
 
     return initApp(options)
-    .then(app => loadRoutes(app, `${__dirname}/routes`))
+    .then((app) => loadRoutes(app, `${__dirname}/routes`))
```

Comment:
So, we really have to decide whether to enforce parentheses in simple statements such as this one. Yes / no / ignore ?

Reply:
Out of 131 instances only 1 did not use parentheses. I figured it's safe to change this line and not make an exception. You guys let me know if you always want to ignore this. I already told you my preference.